chipsec/module_common.py

```diff
@@ -36,8 +43,58 @@ class ModuleResult:
     INFORMATION = 5
     NOTAPPLICABLE = 6
     ERROR = -1
-
-
+    # -------------------------------------------------------
+    class status(Enum):
```

Comment:
Question: will the status be exclusive (single bit set) or can multiple bits be set.

Reply:
Multiple bits can be set